web/src/screens/TestBenchScreen.tsx

```diff
@@ -13,15 +13,24 @@ export const TestBenchScreen: FC = () => {
   const [testBenchDump, setTestBenchDump] = useState<TestBenchReportData | null>(null);
 
   useEffect(() => {
+    const source = Axios.CancelToken.source();
+
     api
-      .runTestBench(appContext.selectedSID, "web_ui", 5)
+      .runTestBench(appContext.selectedSID, source.token, "web_ui", 5)
       .then((response: AxiosResponse<TestBenchReportData | null>) => {
         setTestBenchDump(response.data);
         setRequestSuccess(true);
       })
       .catch((err: AxiosError) => {
-        setRequestSuccess(false);
+        if (!Axios.isCancel(err)) {
+          setRequestSuccess(false);
```

Comment:
Not really, we shouldn't set state if request was cancelled, because component is unmounted by this time. It would cause the rerender and warning after all. The whole request cancellation thing was about excluding set changing after component unmounting.

Reply:
ok